app/helpers/request_info_helper.rb

```diff
@@ -1,9 +1,25 @@
 module RequestInfoHelper
   private
 
+  def request_info_tabs_types
+    {
+      :requester   => _('Requester'),
+      :purpose     => _('Purpose'),
+      :catalog     => _('Catalog'),
+      :environment => _('Environment'),
+      :properties  => _('Properties'),
+      :customize   => _('Customize'),
+      :schedule    => _('Schedule'),
+    }
+  end
+
+  def request_info_tab_label(label)
+    label ? request_info_tabs_types[label.downcase!.to_sym] : ''
+  end
```

Comment:
Minor, but this might be better / more efficient like:

```suggestion
  REQUEST_INFO_TAB_TYPES = {
    :requester   => N_('Requester'),
    :purpose     => N_('Purpose'),
    :catalog     => N_('Catalog'),
    :environment =>N _('Environment'),
    :properties  => N_('Properties'),
    :customize   => N_('Customize'),
    :schedule    => N_('Schedule'),
  }.freeze

  def request_info_tab_label(label)
    label ? _(REQUEST_INFO_TAB_TYPES[label.downcase.to_sym]) : ""
  end
```

Reply:
Also, we should use `downcase` instead of `downcase!` as we shouldn't modify parameters that were given to us.